docs/src/main/sphinx/sql/explain.rst

```diff
@@ -84,6 +84,272 @@ Logical plan::
                               Estimates: {rows: 25 (225B), cpu: 225, memory: 0B, network: 0B}
                               regionkey := tpch:regionkey
 
+EXPLAIN (TYPE LOGICAL, FORMAT JSON)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning:: This is an experimental feature thus output is not guaranteed to be backward compatible across Trino versions.
```

Comment:
We shouldn't say that the feature is experimental, just that `serialization compatibility is not guaranteed across versions`